draft-ietf-quic-http.md

```diff
@@ -912,10 +912,20 @@ Endpoints that set low values for the QUIC transport parameters
 chance that the remote peer reaches the limit early and becomes blocked. In
 particular, the value chosen for `initial_max_uni_streams` should consider that
 remote peers may wish to exercise reserved stream behaviour ({{stream-grease}}).
-To reduce the likelihood of blocking, both clients and servers SHOULD send a
-value of three or greater for the QUIC transport parameter
-`initial_max_uni_streams`, and a value of 1,024 or greater for the QUIC
-transport parameter `initial_max_stream_data_uni`.
+To reduce the likelihood of blocking, both clients and servers MUST allow
+the peer to create one unidirectional stream for the HTTP control stream plus
```

Comment:
Maybe it would be better to state "create _at least_", to avoid the confusion that the endpoints are expected to allow creation *exactly* three streams.

Reply:
Updated, thanks!